<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>417</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>200</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Traversor2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo4">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>Vertex</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Edge</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Face</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo5">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <item>
         <property name="text">
          <string>incident to</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>adjacent by</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo6">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Vertex</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Edge</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Face</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="explodeSlider">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkTopo">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string>show topo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
